test/passing/doc_comments.mli.ref

```diff
@@ -0,0 +1,331 @@
+    the hash of the root chunk node as it is the case for {{!AO} AO}. *)
+
+(** This is verbatim:
+
+    {v
+   o  o
+  /\  /\
+  /\  /\
+    v}
+
+    This is preformated code:
+
+    {[
+      let verbatim s =
+        s |> String.split_lines |> List.map ~f:String.strip
+        |> fun s -> list s "@," Fmt.str
+    ]} *)
+
+(** Lists:
+
+    list with short lines:
+
+    - x
+    - y
+    - z
+
+    list with long lines:
+
+    - xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx
+      xxx xxx xxx xxx xxx xxx xxx
+    - yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy
+      yyy yyy yyy yyy yyy yyy yyy
+    - zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz
+      zzz zzz zzz zzz zzz zzz zzz
+
+    enumerated list with long lines:
+
+    + xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx xxx
+      xxx xxx xxx xxx xxx xxx xxx
+    + yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy yyy
+      yyy yyy yyy yyy yyy yyy yyy
+    + zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz zzz
+      zzz zzz zzz zzz zzz zzz zzz
+
+    list with sub lists:
+
+    {ul
+     {- xxx
+
+        - a
+        - b
+        - c
+     }
+     {- yyy
+
+        + a
+        + b
+        + c
+     }
+    } *)
+
+(** {{:https://github.com/} Github} *)
+
+(** {:https://github.com/} *)
+
+(** An array index offset: [exp1\[exp2\]] *)
+
+(** to extend \{foo syntax *)
+
+(** The different forms of references in \@see tags. *)
+
+(** Printf groff string for the \@before information. *)
+
+(** [a]'c [b]'s [c]'c *)
+
+(** return true if [\gamma(lhs) \subseteq \gamma(rhs)] *)
+
+(** Composition of functions: [(f >> g) x] is exactly equivalent to
+    [g (f (x))]. Left associative. *)
+
+(** [†] [Struct_rec] is *)
+
+(** for [Global]s *)
+
+(** generic command: ∀xs.[foot]-[post] *)
+
+(** A *)
+val foo : int -> unit
+(** B *)
+
+(** C *)
+
+(** A *)
+val foo : int -> unit
+(** B *)
+
+module Foo : sig
+  (** A *)
+  val foo : int -> unit
+  (** B *)
+
+  (** C *)
+
+  (** A *)
+  val foo : int -> unit
+  (** B *)
+end
+
+(** [\[ \] \[\] \]] *)
+
+(** \{ \} \[ \] \@ \@ *)
+
+(** @canonical Foo *)
+
+(** @canonical Module.Foo.Bar *)
+
+(** {v a v} *)
+
+(** {[ b ]} *)
+
+(** - Odoc don't parse
+
+    multiple paragraph in a list *)
+
+(** {ul
+     {- Abc
+
+        Def
+     }
+     {- Hij }
+     {- Klm
+
+        {ul
+         {- Nop
+
+            Qrs
+         }
+         {- Tuv }
+        }
+     }
+    } *)
+
+(** - {v
+    Abc
+    def
+      v}
+    - {[
+        A
+          B
+      ]} *)
+
+(** Code block
+
+    {[ Single line ]}
+    {[
+      Multi
+      line
+    ]}
+    {[
+      Multi
+      line
+        with
+          indentation
+    ]}
+    {[
+      Single long line HAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+    ]}
+    {[
+      With empty
+
+      line
+    ]}
+    {[
+      First line
+            on the same line
+            as opening
```

Comment:
This requires a fix to Odoc. https://github.com/ocaml/odoc/pull/370

Reply:
It is now merged but will be fixed in OCamlformat only after the next release of Odoc.